src/GraphQL/Types/Collections/SchemaTypes.cs

```diff
@@ -656,7 +694,7 @@ private IGraphType AddTypeIfNotRegistered(Type type, TypeCollectionContext conte
                 else if (_builtInCustomScalars.TryGetValue(namedType, out var builtInCustomScalar))
                 {
                     foundType = builtInCustomScalar;
-                    AddType(foundType, _context);
+                    AddType(foundType, context);
```

Comment:
@Shane32 I changed this accidentally. Looks like a bug thought. 🤔

Reply:
On the other side - `_context` is a field and this is the only usage outside of `Initialize` method.